plasma_framework/contracts/mocks/exits/payment/PaymentStandardExitableMock.sol

```diff
@@ -2,18 +2,36 @@ pragma solidity ^0.5.0;
 pragma experimental ABIEncoderV2;
 
 import "../../../src/exits/payment/PaymentStandardExitable.sol";
+import "../../../src/framework/PlasmaFramework.sol";
+import "../../../src/vaults/EthVault.sol";
+import "../../../src/vaults/Erc20Vault.sol";
 
 contract PaymentStandardExitableMock is PaymentStandardExitable {
-    constructor(address _framework) public PaymentStandardExitable(_framework) {}
+    PlasmaFramework private framework;
 
-    // to make contract not abstract
+    constructor(PlasmaFramework _framework, EthVault _ethVault, Erc20Vault _erc20Vault)
+        public
+        PaymentStandardExitable(_framework, _ethVault, _erc20Vault)
+    {
+        framework = _framework;
+    }
+
+    /** override to make this non abstract contract */
     function processExit(uint256 _exitId) external {}
 
+    function processStandardExit(uint256 _exitId) external {
+        PaymentStandardExitable._processStandardExit(_exitId);
```

Comment:
For this one, I am not 100% sure what do we like to take as our convention actually.

Because we do have such style in our [existing RootChain code](https://github.com/omisego/plasma-contracts/blob/master/contracts/RootChain.sol#L287), also, [Openzepplin code](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC20/ERC20.sol#L153) has the same style as well.

But on the other hand, `PaymentStandardExitable.processStandardExit` just looks better than `PaymentStandardExitable._processStandardExit` IMO, so I will probably make the change here.

Reply:
But in this case does the underscore indicate a private function? 
The usage (if we translate it to OOP) would mean sth like `protected`.

Reply:
Added an issue for style guide: https://github.com/omisego/plasma-contracts/issues/191